### PR TITLE
feat: Port changes from upstream version stream and adapt the pipeline files

### PR DESCRIPTION
**Pull request by ccojocar**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -13,12 +12,10 @@ approvers:
...
 - jstrachan
-- rajdavies
 - ccojocar
 - garethjevans
 - pmuir
 - daveconde
 - warrenbailey
 - cagiti
 - abayer
-- macox
```
Comment (abayer) [inline OWNERS]: While we're here, would make sense to add Hardy and Daniel.

Comment (ccojocar) [reply OWNERS]: yeah

Comment (ccojocar) [reply OWNERS]: Done

**Pull request merged**